src/BladeError/BladeErrorInterface.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace HelsingborgStad\BladeError;
+
+use Throwable;
+
+interface BladeErrorInterface
+{
+    public function __construct(Throwable $e);
```

Comment:
I would suggest to add a getThrowable():Throwable to the interface instead of defining the constructor params in the interface to avoid limiting possible implementaions.